connect_ext_test.go

```diff
@@ -93,8 +93,9 @@ func TestServer(t *testing.T) {
 			assert.Equal(t, connect.CodeOf(err), connect.CodeInvalidArgument)
 		})
 		t.Run("ping_timout", func(t *testing.T) {
-			ctx, cancel := context.WithTimeout(context.Background(), time.Nanosecond)
+			ctx, cancel := context.WithTimeout(context.Background(), time.Millisecond)
```

Comment:
Tests were failing on Windows - changed to use a longer timeout and explicitly wait prior to making the call.